static-site/src/components/CardsV2/card-component.jsx

```diff
@@ -0,0 +1,205 @@
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'flex-start',
+        gap: '10px',
+        alignItems: 'center',
+        color: '#4F4B50',
+      }}>
+        {outer && Logo(data.name)}
+        <Name name={data.name} url={data.url}/>
+        <SparkLine versions={data.versions || []} onClick={()=>{setDetails(details==='versions'?'':'versions')}}/>
+      </div>
+
+      <IconRow data={data} details={details} setDetails={setDetails}/>
+
+      <Details data={data} view={details}/>
+
+      {(data.children) && (data.children.length!==0) && /* !data.collapsed && */
+        data.children.map((card) => <Card key={card.name} data={card}/>)
+      }
+    </div>
+  )
+}
+
+
+function IconRow({data, details, setDetails}) {
+
+  const setDetailsWrapper = (value) => {
+    if (value===details) setDetails('')
+    else setDetails(value)
+  }
+  return (
+    <>
+      <div style={{
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'flex-start',
+        color: "#A49DA7",
+        fontSize: "1.3rem",
+        gap: '15px',
+      }}>
+        <LinkedPathogenPageIcon name={data.name}/>
+        { data.nDatasets > 1 && (
+          <IconContainer
+            description="Total number of distinct resource collections (e.g. datasets)"
+            Icon={MdOutlineShare}
+            text={data.nDatasets}
+          />
+        )}
+        <IconContainer
+          description="Total number of available versions"
+          Icon={MdCached}
+          text={data.nVersions}
+          // handleClick={() => setDetailsWrapper('versions')}
+        />
+        <div>
+          {`Last updated: ${data.lastUpdated}`}
+        </div>
+      </div>
+    </>
+  )
+}
+
+function IconContainer({description, Icon, text, handleClick=undefined, selected=false}) {
+  const iconProps = {size: "1.2em"};
+  if (selected) iconProps.color = '#4F4B50';
+  const hasOnClick = typeof handleClick === 'function';
+  return (
+    <div
+      style={{display: 'flex', alignItems: 'center', gap: '3px', cursor: hasOnClick?'pointer':'auto'}}
+      data-tooltip-id="iconTooltip"
+      data-tooltip-content={description}
+      data-tooltip-place="top"
+      onClick={hasOnClick ? handleClick : ()=>{}}
+    >
+      <Icon {...iconProps}/>
+      {text}
+    </div>
+  )
+}
+
+function LinkedPathogenPageIcon({name}) {
+  const pages = {
+    flu: "/influenza",
+    ncov: "/sars-cov-2",
+  }
+  const p1 = name.split('/')[0]
+  if (!pages[p1]) return null;
+
+  return (
+    <a
+      style={{display: 'flex', alignItems: 'center', gap: '3px', cursor: 'pointer', color: 'inherit', fontWeight: '300'}}
+      data-tooltip-id="iconTooltip" data-tooltip-place="top"
+      data-tooltip-content="Click to view associated pathogen page"
+      href={`https://nextstrain.org${pages[p1]}`}  target="_blank" rel="noreferrer"
+    >
+      <MdCoronavirus size={"1.2em"}/>
+      {pages[p1]}
+    </a>
+  )
+}
+
+function Details({data, view}) {
```

Comment:
[Potentially blocking] The ability to show more detailed information about versions below a card has been disabled here - partly as it requires a bit of polish, and partly as it used d3 to manipulate the DOM and d3 + gatsby = 😡.

Because we group datesamped datasets with the non-datestamped versions (i.e. `.../6m/2023-01-01` is shown as a version of `.../6m`, rather than its own dataset), there is currently no way to actually access this datestamped dataset. These were exposed in the (now removed) table, so this is arguably a downgrade. Although that table was incomplete, so...